### ISSUE TITLE
May I ask about the configs of pre-training? For example, did you use dropout?

### ISSUE THREAD

**Issue by joytianya** [young-geng/EasyLM#51]
May I ask about the configs of pre-training? For example, did you use dropout?

**Comment by young-geng** [comment]
If you asking about OpenLLaMA, we follow the exact same configuration of LLaMA, so you can find all the hyperparameters in the LLaMA paper.

**Comment by joytianya** [comment]
I didn't find dropout in the paper. May I ask if llama has used dropout?
